contrib/svt-av1/module.defs

```diff
@@ -15,6 +15,16 @@ SVT-AV1.CONFIGURE.static      =
 SVT-AV1.CONFIGURE.shared      = -DBUILD_SHARED_LIBS=OFF
 SVT-AV1.CONFIGURE.extra       = -DBUILD_DEC=OFF
 
+ifeq ($(GCC.O),$(filter $(GCC.O),size size-aggressive))
+    SVT-AV1.CONFIGURE.extra += -DCMAKE_BUILD_TYPE=MinSizeRel
+else
+    ifneq (none,$(X265_8.GCC.g))
```

Comment:
Copy paste typo here

Reply:
Ah yes. This was a stupid copy paste typo. 🤦 I fixed it.